src/infrastructure/implementations/crux-messenger.ts

```diff
@@ -221,3 +221,29 @@ export const cruxPaymentProtocol: IMessageSchema[] = [{
             .required().min(36).max(36),
     }),
 }];
+
+export const keyManagementProtocol: IMessageSchema[] = [
+    {
+        messageType: "KEY_MANAGER_REQUEST",
+        schema: Joi.object({
+            args: Joi.array()
+                .required(),
+
+            invocationId: Joi.string()
+                .required().min(36).max(36),
+
+            method: Joi.string().valid("signWebToken", "getPubKey", "deriveSharedSecret", "decryptMessage")
```

Comment:
👍 on defining method names in protocol itself

Reply:
thanks